<launch>
	<node pkg="move_base" type="move_base" respawn="false" name="move_base" output="screen" clear_params="true">
        <remap from="odom" to="odom" />
        <remap from="cmd_vel" to="cmd_vel" />
        <remap from="map" to="/map" />

    	<param name="footprint_padding" value="0.01" />
    	<param name="controller_frequency" value="10.0" />
    	<param name="controller_patience" value="100.0" />
    	<param name="planner_frequency" value="2.0" />

    	<rosparam file="$(find morse_basic_example_nav)/costmap_common_params.yaml" command="load" ns="global_costmap" />
    	<rosparam file="$(find morse_basic_example_nav)/costmap_common_params.yaml" command="load" ns="local_costmap" />
    	<rosparam file="$(find morse_basic_example_nav)/local_costmap_params.yaml" command="load" />
    	<rosparam file="$(find morse_basic_example_nav)/global_costmap_params.yaml" command="load" />
    	<param name="base_local_planner" value="dwa_local_planner/DWAPlannerROS" />
    	<rosparam file="$(find morse_basic_example_nav)/dwa_planner_ros.yaml" command="load" />
        
        <param name="~/global_costmap/robot_base_frame" value="base_link" />
        <param name="~/local_costmap/robot_base_frame" value="base_link" />
        <param name="~/local_costmap/global_frame" value="odom" />
	</node>
    
    <!--- Run AMCL -->
    <node name="amcl" pkg="amcl" type="amcl">
        <remap from="scan" to="base_scan" />
        <remap from="base_scan" to="/base_link" />
    </node>

</launch>
